animepahe-dl.sh

```diff
@@ -62,6 +67,13 @@ set_args() {
             r)
                 _ANIME_RESOLUTION="$OPTARG"
                 ;;
+            t)
+                _PARALLEL_RUN="true"
+                _PARALLEL_JOBS="$OPTARG"
```

Comment:
Can remove variable `_PARALLEL_RUN`, use only `_PARALLEL_JOBS`, and check if it's set or not `${_PARALLEL_JOBS:-}`

Reply:
Resolved Snippet:
```
    if [[ ! -z ${_PARALLEL_JOBS:-} ]]; then
       _XXD="$(command -v xxd)" || command_not_found "xxd"
       _OPENSSL="$(command -v openssl)" || command_not_found "openssl"
    fi
    
```

Reply:
Resolved:
https://github.com/KevCui/animepahe-dl/pull/16#discussion_r560196873
Other edit in source:

```
        if [[ ! -z ${_PARALLEL_JOBS:-} ]]; then

```